templates/io-package.json.ts

```diff
@@ -66,8 +66,8 @@ export = (async answers => {
 		"main": "${useTypeScript ? "build/" : ""}main.js",
 		"icon": "${getIconName(answers)}",
 		"enabled": true,
-		"extIcon": "https://raw.githubusercontent.com/${answers.authorGithub}/ioBroker.${answers.adapterName}/master/admin/${getIconName(answers)}",
-		"readme": "https://github.com/${answers.authorGithub}/ioBroker.${answers.adapterName}/blob/master/README.md",
+		"extIcon": "https://raw.githubusercontent.com/${answers.authorGithub}/ioBroker.${answers.adapterName}/main/admin/${getIconName(answers)}",
+		"readme": "https://github.com/${answers.authorGithub}/ioBroker.${answers.adapterName}/blob/main/README.md",
```

Comment:
Can we make this configurable - can be an expert question.

I'm being selfish, I never got used to `main`:
![grafik](https://user-images.githubusercontent.com/17641229/132585321-610bab2a-296c-423d-832e-0b0c2b1a2d0d.png)

Reply:
What abou this?

Reply:
Done.